apps/dashr-drug-discovery/app.R

```diff
@@ -0,0 +1,262 @@
+      htmlTd(htmlA(href=toString(df_subset$PAGE[[i]]), children = "Datasheet")))
+    )
+  }
+  return(table)
+}
+#for dccDropDown options
+drug_names <- unique(df$NAME)
+drug_options <- list()
+for (i in 1:length(drug_names)){
+  drug_options[[i]] <- list(label = drug_names[i], value = drug_names[i])
+}
+
+FIGURE = three_d #initial figure
+STARTING_DRUG = 'Levobupivacaine'
+#get img url from csv, initalized with starting drug
+DRUG_DESCRIPTION = df$DESC[df$NAME == STARTING_DRUG]
+#get desc from csv, initalized with starting drug
+DRUG_IMG = df$IMG_URL[df$NAME == STARTING_DRUG]
+
+app$layout(
+  htmlDiv(list(
+    htmlDiv(list(
+      #Dash icon
+      htmlImg(src="https://s3-us-west-1.amazonaws.com/plotly-tutorials/logo/new-branding/dash-logo-by-plotly-stripe.png",
+              style = list(height = '60px', float ='left', position = 'relative', right = '10px'))
+    ),className = 'app-banner'),
+    
+    htmlDiv(list(
+      htmlDiv(list(
+        htmlDiv(list(
+          #title + subtitles
+          htmlBr(),
+          title,
+          subtitle1_bold,
+          subtitle1,
+          htmlBr(),
+          subtitle2_bold,
+          subtitle2
+        ))
+      ), className = 'app__header'),
+      htmlDiv(list(
+        dccDropdown(
+          id = 'chem_dropdown',
+          multi = TRUE,
+          value = STARTING_DRUG, 
+          options = drug_options
+        )
+      ), className = "app__dropdown"),
+      htmlDiv(list(
+        htmlDiv(list(
+          dccRadioItems(
+            id = 'charts_radio',
+            options = list(list(label = '3D Scatter', value = 'scatter3d'),
+                           list(label = '2D Scatter', value = 'scatter'),
+                           list(label = '2D Histogram', value = 'histogram2d')),
+            labelStyle = list(display='inline'), 
+            labelClassName = "radio__labels",
+            inputClassName = "radio__input",
+            value = 'scatter3d',
+            className = 'radio__group'
+          ),
+          dccGraph(
+            id = 'clickable-graph',
+            hoverData = list(points = list(), range=NULL),
+            figure = FIGURE
+          )
+        ), className= 'two-thirds column'),
+        htmlDiv(list(
+          htmlImg(
+            src = DRUG_IMG , id='chem_img'
+            ),
+          htmlA(
+            children=STARTING_DRUG, id='chem_name', href = "https://www.drugbank.ca/drugs/DB01002",target='_blank'
+          ),
+          htmlP(
+            children=DRUG_DESCRIPTION, id='chem_desc'
+          )
+        ),className = "chem__desc__container")
+      ), className = "container card app__content bg-white"),
+      htmlDiv(list(
+        htmlTable(
+          id = 'table-element',
+          children=make_dash_table(STARTING_DRUG),
+          className = "table__container"
+        )
+      ), className = "container bg-white p-0")
+    ),className = "app__container")
+  ))
+)
+
+
+#####CALL BACKS#####
```

Comment:
Maybe rename `CALL BACKS` to `CALLBACKS`? I realize it's a comment, being nit picky here.